### PR TITLE
Refactor logging

### PR DESCRIPTION
**Pull request by vladimyr**
This PR implements logger registry with global `.enabled` property & log level setting (`LOG_LEVEL`). i.e. if you leave default env vars you won't see sql queries in your console due to `LOG_LEVEL` set to `info` by default and `database` module uses `debug` instead. ☝️

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -59,7 +59,7 @@ ENABLE_DEFAULT_SCHEMA=1
 PREVIEW_URL=https://boutique.localhost/api/v1/preview/
 
 # Logger
-LOG_LEVEL='info'
+LOG_LEVEL=info
```
Comment (vladimyr) [inline .env.example:63]: Everything inside dotenv is a string by default.

File: server/script/addAdmin.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
 'use strict';
 
+require('dotenv').config();
+require('../shared/logger').enabled = false;
```
Comment (vladimyr) [inline server/script/addAdmin.js:4]: This prevents sql queries to get logged.

File: server/shared/database/config.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-const logger = require('../logger');
+const { createLogger, Level } = require('../logger');
 
 const isProduction = process.env.NODE_ENV === 'production';
+const logger = createLogger('db', { level: Level.DEBUG });
```
Comment (vladimyr) [inline server/shared/database/config.js:6]: Database module uses `debug` log level.

File: server/shared/logger.js [hunk 4]
```diff
@@ -1,16 +1,57 @@
...
+  _emit(record, noemit) {
+    return super._emit(record, !createLogger.enabled || noemit);
+  }
+}
+
+function createLogger(name, options = {}) {
+  name = [pkg.name, name].filter(Boolean).join(':');
+  const serializers = { ...Bunyan.stdSerializers, ...options.serializers };
+  if (!loggers[name]) {
+    options.level = process.env.LOG_LEVEL || options.level;
```
Comment (vladimyr) [inline server/shared/logger.js:29]: Unless specified, the default logger level matches `LOG_LEVEL` env var to make sure it gets printed to the stdout.

File: server/shared/logger.js [hunk 5]
```diff
@@ -1,16 +1,57 @@
...
+module.exports = createLogger;
+
+function getLevels(Logger) {
+  const { levelFromName: levels } = Logger;
+  return Object.keys(levels).reduce((acc, name) => {
+    return Object.assign(acc, { [name.toUpperCase()]: levels[name] });
+  }, {});
+}
+
+function getOutputStream(loggerStream) {
```
Comment (underscope) [inline server/shared/logger.js]: processOutputStream / processLogOutput 🤔? (Some doc would be nice as well explaining whys ;) )

File: server/shared/logger.js [hunk 6]
```diff
@@ -1,16 +1,57 @@
...
+  return loggers[name];
+}
+Object.assign(createLogger, Logger, { enabled: true, createLogger, Level });
+
+module.exports = createLogger;
+
+function getLevels(Logger) {
+  const { levelFromName: levels } = Logger;
+  return Object.keys(levels).reduce((acc, name) => {
+    return Object.assign(acc, { [name.toUpperCase()]: levels[name] });
```
Comment (underscope) [inline server/shared/logger.js]: Spread might be cleaner?

File: server/shared/logger.js [hunk 7]
```diff
@@ -1,16 +1,57 @@
...
+function createLogger(name, options = {}) {
+  name = [pkg.name, name].filter(Boolean).join(':');
+  const serializers = { ...Bunyan.stdSerializers, ...options.serializers };
+  if (!loggers[name]) {
+    options.level = process.env.LOG_LEVEL || options.level;
+    loggers[name] = new Logger({ ...options, name, serializers });
+  }
+  return loggers[name];
+}
+Object.assign(createLogger, Logger, { enabled: true, createLogger, Level });
```
Comment (underscope) [inline server/shared/logger.js]: Why so convoluted 🤔?

**Pull request merged**